region/norfair/east/Bubble Mountain.json

```diff
@@ -1110,6 +1110,194 @@
         "canTrickyGrappleJump"
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "Side Platform Cross Room Jump",
+      "entranceCondition": {
+        "comeInWithSidePlatform": {
+          "platforms": [
+            {
+              "minHeight": 1,
+              "maxHeight": 1,
+              "minTiles": 13,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [
+                {"or": [
+                  "HiJump",
+                  {"and": [
+                    "canTrickyDashJump",
+                    "canTrickySpringBallJump"
+                  ]}
+                ]},
+                {"or": [
+                  "canTrickyDodgeEnemies",
+                  {"enemyDamage": {"enemy": "Waver", "type": "contact", "hits": 1}}
+                ]}
+              ],
+              "note": ["This applies to Warehouse Entrance."]
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 11.4375,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [
+                {"or": [
+                  {"and": [
+                    "HiJump",
```

Comment:
Leaving ridley tank room feels like Collision Oscillator.  Something is making it harder than it should be.

Reply:
Key things:
1) Don't back into the corner; just press against the wall, turn around, and run.
2) Aim down immediately after jumping.

With this, there's a two-frame window for the jump, with extra run speeds $2.B or $2.C. If you back into the corner, then $2.B won't work because you'll bonk the ceiling of the door frame. And if you don't aim down fast enough, $2.C won't work because you'll catch on the floor of the door frame. So if you were to back into the corner and try to spin jump out (without aiming down), then it just wouldn't work at all regardless of the jump timing.

Added some notes to the Ridley Tank Room setup now.